packages/dashboard-core-plugins/src/WidgetLoaderPlugin.tsx

```diff
@@ -72,7 +72,6 @@ export function WrapWidgetPlugin(
             {...props}
           />
         )}
-        )
```

Comment:
How'd this not throw an error before?

Reply:
This was actually displaying a parenthesis after the component if it was auto wrapped